tests/includes/API/Test_Hook_Isolation.php

```
@@ -194,8 +194,22 @@ public function test_route_map_only_contains_wcpos_routes(): void {
 		$property->setAccessible( true );
 		$route_map = $property->getValue( $api );
 
+		$prefixes = array_map(
+			static function ( string $ns ): string {
+				return '/' . $ns . '/';
+			},
+			\WCPOS\WooCommercePOS\API::ROUTE_NAMESPACES
+		);
+
 		foreach ( $route_map as $route => $key ) {
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$key'. (undefined)

(UnusedLocalVariable)